ios/LayoutReanimation/REASharedTransitionManager.m

```diff
@@ -497,4 +605,33 @@ - (void)clearAllSharedConfigsForViewTag:(NSNumber *)viewTag
   }
 }
 
+- (void)cancelAnimation:(NSNumber *)viewTag
+{
+  _cancelLayoutAnimation(viewTag, @"sharedTransition", YES, YES);
+}
+
+- (void)disableCleaningForViewTag:(NSNumber *)viewTag
+{
+  NSNumber *counter = _disableCleaningForView[viewTag];
+  if (counter != nil) {
+    _disableCleaningForView[viewTag] = @([counter intValue] + 1);
```

Comment:
Since NSNumber is an object, can't we just increment it directly? Is it read-only?

Reply:
Unfortunately `NSNumber` is an immutable object type.